tests/integration/components/paper/file-input-test.js

```diff
@@ -0,0 +1,144 @@
+import { render, find }       from '@ember/test-helpers';
+
+import hbs                    from 'htmlbars-inline-precompile';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test }       from 'qunit';
+
+module('Integration | Component | {{paper/file-input}}', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    await render(hbs`{{paper/file-input}}`);
+    assert.ok(find('button'), 'shows a button');
+    assert.ok(find('button > i.fa.fa-upload'), 'shows default icon in button');
+    assert.ok(find('input[type=file]'), 'Has (hidden) file input');
+    assert.equal(this.element.textContent.trim(), 'No file selected',
+      'shows default message');
+  });
+
+  test('sets `accept` attribute on input', async function(assert) {
```

Comment:
Is it possibile to allow multiple file types? Just a question, don't have to implement it for now.

Reply:
Yes it is, it works just the same as the normal `<input>` [accept attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#attr-accept)